docs/content/en/docs/snippets/tasks/install.md

```diff
@@ -1,7 +1,28 @@
 
 ## Install version 0.6.0 and above
 
-In version 0.6.0 and later, you can install the Lifecycle Toolkit using the current release manifest:
+In version 0.6.0 and later, you can install the Lifecycle Toolkit using helm charts:
```

Comment:
```suggestion
In version 0.6.0 and later, you can install the Lifecycle Toolkit using Helm charts:
```

Reply:
I would start the section by telling the user that we have two options
```suggestion
In version 0.6.0 and later, you can install the Lifecycle Toolkit using Helm charts or manifests.

For installing the Lifecycle Toolkit via Helm chart:
```